ui/main/src/app/services/userSettings/UserSettingsService.ts

```
@@ -8,26 +8,26 @@
 
-    public static setSettingsServer(settingsServer: SettingsServer) {
-        CurrentUserStore.getCurrentUserLogin().subscribe((id) => (SettingsService.userId = id));
-        SettingsService.settingsServer = settingsServer;
+    public static setUserSettingsServer(settingsServer: UserSettingsServer) {
+        CurrentUserStore.getCurrentUserLogin().subscribe((id) => (UserSettingsService.userId = id));
+        UserSettingsService.userSettingsServer = settingsServer;
     }
 
     static getUserSettings(): Observable<any> {
-        return SettingsService.settingsServer.getUserSettings(this.userId);
+        return UserSettingsService.userSettingsServer.getUserSettings(this.userId);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)